tests/test_errors.py

```diff
@@ -31,34 +31,34 @@
 #         thermal_network.producers['node_id'] = np.float(thermal_network.producers['node_id'])
```

Comment:
Add a note in the code that the assertions that let these tests fail have yet to be implemented.